### PR TITLE
Precision about HTTP/2 handling

### PR DESCRIPTION
**Pull request by Soulou**

### CODE REVIEW

File: _posts/platform/internals/2000-01-01-routing.md [hunk 1]
```diff
@@ -55,6 +55,15 @@ HTTP/2 is enabled by default for all applications. If the client supports it
 and is reaching the servers with HTTPS, the connection will be upgraded to
 HTTP/2 by our routing nodes.
 
+The traffic between the routing nodes and the application containers is then done
+using HTTP/1.1, so your application should **not** expect HTTP/2 traffic.
```
Comment (EtienneM) [inline _posts/platform/internals/2000-01-01-routing.md:59]:
```suggestion
using HTTP/1.1. Hence, your application should **not** expect HTTP/2 traffic.
```

File: _posts/platform/internals/2000-01-01-routing.md [hunk 2]
```diff
@@ -55,6 +55,15 @@ HTTP/2 is enabled by default for all applications. If the client supports it
...
+
+Performance improvement due to HTTP/2 would still be present as the protocol
+has been designed to be faster over slow connections (ie. mobile, xDSL) and these
```
Comment (EtienneM) [inline _posts/platform/internals/2000-01-01-routing.md:62]:
```suggestion
has been designed to be faster over slow connections (i.e. mobile, xDSL) and these
```

File: _posts/platform/internals/2000-01-01-routing.md [hunk 3]
```diff
@@ -55,6 +55,15 @@ HTTP/2 is enabled by default for all applications. If the client supports it
...
+terminals would use HTTP/2. Then, multiplexed requests are achieved in parallel
+using HTTP/1.1 to the containers, as these are in the same network, HTTP/2 would
```
Comment (EtienneM) [inline _posts/platform/internals/2000-01-01-routing.md:64]:
```suggestion
using HTTP/1.1 to the containers. As these are in the same network, HTTP/2 would
```

**Comment by cyrille-sg** [comment]
I known what this is about so the explanation is clear. But I am not sure if I did not knew, it will be clear.

What do you think to add a small diagram to the explanation (and more generally, start to use diagrams)? ([source](https://app.diagrams.net/?lightbox=1&highlight=0000ff&edit=_blank&layers=1&nav=1&title=Untitled%20Diagram.svg#R5Vldk6I6EP01Pi4FRBQfR539qt2qqZ3ZunufrAgRUxsIFeKo%2B%2BtvB4ICySzuXHWtmnkYyUnSkHO60x0YoFm6%2ByBwvv7KY8IGvhvvBmg%2B8H0PBS78KGRfIWEQVkAiaKwHHYFH%2BotoUM9LNjQmRWug5JxJmrfBiGcZiWQLw0LwbXvYirP2XXOcEAN4jDAz0X9oLNca9Vz32PGR0GQt6%2FXpjiWOfiaCbzJ9v4GPVuVf1Z3i2pYeX6xxzLcNCN0P0ExwLqurdDcjTHFb01bNe%2F9C7%2BG5BcnkKRM%2Bf%2F6%2B5u%2Fvv31lQZBF755wehe9Q9rMM2YbUq%2BjfFq5rxkq10iUFXeApts1leQxx5Hq3YJPALaWKYOWB5crytiMMy7KuSgOSBgPAS%2Bk4D9Joyf0l2g0gh79AERIsntxad6BMHBEwlMixR6G1BMmmmPtg6gWb3tUtB6ybmjpIQ1i7UTJwfSRSLjQXP4Br%2F7kwrxiEq4iG6%2BjKCTL1Xl49cN%2BXj3%2FqsSe4K85p5ks7xtMB8EcqMBFXm0dK7pThDeJxYwmGVxHwAkRitE6TpUqMS7WB4loWu4l9e%2BcpgmsgdEl%2FMeMURIvJGGkoAUAEU%2FzDVhc4EzBIqUZZqrjgYiCw%2FVipkc4xXNyJr3crl4jxw0MyUZjZzw0RZt4Tngh1dBlo2G1IqPIGg3xeLJ03fOwOzTYNaMBBl11m%2BnnlW8koxlwUqdP97c8vcRsKxAUjRRS6Be8JOyBF1RSrmJoyaXkaWPAnQ4uyXNbqDUV5ZnUxYHn1229CtcWwhCmuepMd4kqTBy8LYaOIAXfiIh8itTzTKFZXbVHASQxcCIW%2Fnk8I%2FA7nuH5tribBM7EEnYH9OzuMfzTzXLqOq4HDz47%2FKoN1J%2BVPZOwr0f%2FNnpqa2aPXwFt0IaNTdArbfqBBbRh4%2B69S8Cz3LuL%2BRbQatJyb7fzkGUm6o2cRki8EETdQDCiqpSUiPtnUinrWaIlokXEvQkEjLI1zYWOExUaMG0Bu3Fl7K%2FstGjciSdkSWKWDBZcap8d9QcSyeI7dRxRJDJcFLRiCQtpwg2RgRGx%2F6E3ubLxr2qAL%2BnmfNfsnO%2BbLSglKCxQCTU%2FUk9i48TTIR6OWFgkRPZtHaZADQECS6arMUEYlvS5%2FRg2VfQdHpTL%2FqbuHHWErXZ5Pat55ukYCjqGhl7HUMWDYah0ksOy%2F0e1ekKCfq3j7Kj80bhuuA20jl6jGvtG4%2FU%2BU5F%2Bqz4ToI7Ur%2FYZ1ON8l%2FaZwPCZ7wXI1fUb2F5l2yWg3KG%2F8PJYM20kZJiqprIlim6CSWkcq8lT8wTVTAMZz4iRATTYruLPkAqgeHbCcVvZoVl328pu%2F2JnUDMdqBdKNEs4oN%2Bq1Pk25UKB36vV4dXadcQKDbFmdf3%2FRkUKkO9Mgl6dxleVyXxhdoypW1HptAL4%2FHq9VMU0y%2BFrilUbboj18enpwaxt3kY8oe6LIXQ40vfUPpdTyHxPqhQCxHujIg2DU0S6ahghM4w%2BKUYzIm9HpNM%2BNpy%2F7ru1PQ%2BZR7kvHOi8LaX6P7edXanuxzebUmcqz6F5%2FF5anbeOH6XR%2FX8%3D) )
http2

**Comment by cyrille-sg** [comment]
> It's great.. How have live without knowing draw.io?

Not sure I understand the question

**Comment by Soulou** [comment]
> > It's great.. How have live without knowing draw.io?
> 
> Not sure I understand the question

If I wasn't typing like a monkey you would have understood (it was rhetorical)

**Comment by ksol** [comment]
What's missing here ?

**Comment by cyrille-sg** [comment]
Nothing FMPOV

**Pull request merged**